tests/smoke-tests/test-polkadot-decoding.ts

```diff
@@ -7,12 +7,14 @@ const pageSize = (process.env.PAGE_SIZE && parseInt(process.env.PAGE_SIZE)) || 5
 describeSmokeSuite("Polkadot API - Storage items", (context) => {
   let atBlockNumber: number = 0;
   let apiAt: ApiDecoration<"promise"> = null;
+  let specVersion: number = 0;
 
   before("Setup api", async function () {
     atBlockNumber = (await context.polkadotApi.rpc.chain.getHeader()).number.toNumber();
     apiAt = await context.polkadotApi.at(
       await context.polkadotApi.rpc.chain.getBlockHash(atBlockNumber)
     );
+    specVersion = await apiAt.runtimeVersion.specVersion.toNumber();
```

Comment:
spec version can be grabbed from Api consts.system.version.SpecVersion which won't require async